hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java

```diff
@@ -126,6 +126,11 @@ private S3ErrorTable() {
   public static final OS3Exception NO_OVERWRITE = new OS3Exception(
       "Conflict", "Cannot overwrite file with directory", HTTP_CONFLICT);
 
+  public static final OS3Exception METADATA_TOO_LARGE = new OS3Exception(
+      "MetadataTooLarge", "Illegal user defined metadata. Combined size " +
+      "exceeds the maximum allowed metadata size of 2KB",
```

Comment:
Please use S3_REQUEST_HEADER_METADATA_SIZE_LIMIT_KB instead of 2KB here.

Reply:
Thank you for the review! I have changed it